incubator/rundeck/templates/deployment.yaml

```diff
@@ -101,14 +101,14 @@ spec:
               containerPort: 4440
           livenessProbe:
             httpGet:  
-              path: /
+              path: /user/login
```

Comment:
Read of it fast but seems that that API needs a valid token in order to grant access. With some hacking on an installed version you can expose that api to publi access, but I think that this could be complex to implement. Meanwhile /user/login page is yet exposed, and when it respond also the entire rundeck stack is ready for respond... I'm according that is not the best solution at all, but it could be a good workaround to make everithing working... the url that actually you are using to check (/ path) redirects to /user/login so this change won't introduce nothing new to the code but will make possible usage on GCE... this seems enough fair for me respect changing the rundeck stack in order to use an API that seems to be present from 3.3 version (actually you are using as default 3.2.8 in the helm chart) .

Reply:
/assign @dwardu89

Reply:
That rundeck API path is not public accessible and hacking on the installed version of rundeck is needed in order to make it accessible. It seems also that it is present only from 3.3 version of rundeck (actually helm chart is defaulting th 3.2.8 version). Changes needed in order to make it work would be potentially problematic... Actually the k8s probes are yet checking /user/login page, but they do it after a redirect from / path: so the proposed change would not be disruptive (I'm removing the redirect that is not useful). This workaround does not change the actual check policy but enable GKE to work properly. This semms fair to me. If then someone want to integrate rundeck's metric api it woul be great but probably this is more than needed

Reply:
okay fair enough, then i'll be okay with it.